src/recordlinker/linking/mpi_service.py

```diff
@@ -21,7 +21,7 @@ def get_block_data(
     blocking keys defined in the algo_config.
     """
     # Create the base query
-    query = session.query(models.Patient)
+    expr = expression.Select(models.Patient.id).distinct()
```

Comment:
Could probably import `select` straight from `sqlalchemy`. (Only mentioning this because I used the `select` somewhere else in the `algorithm_service.py` code). I don't think it matters functionally though.

Reply:
They're both defined in [sqlalchemy.sql.expressions](https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/sql/expression.py), so availability of them isn't an issue.

Looks like the [select() function just invokes Select.__init__() class](https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/sql/_selectable_constructors.py#L451-L494).  TBH, I'm not sure what the point is, I think maybe for some backwards compatibility between the 1.x and 2.x styles?

I'm going to make the change though, since the [style guide](https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#migration-orm-usage) does show examples of the function, but not the class.

Reply:
changed. https://github.com/CDCgov/RecordLinker/pull/66/commits/88a28d5b502171626352501771841637077edc8c

Reply:
Okay sounds good! Yeah I'm hoenstly not sure what the difference is either besides I see `select()` in the current documentation